kokoro/linux/build.sh

```diff
@@ -102,7 +102,7 @@ fi
 echo $(date): Starting ctest...
 if [ $SKIP_TESTS = "False" ]
 then
-  ctest --output-on-failure
+  ctest --output-on-failure -j4
```

Comment:
ctest --output-on-failure `nproc` 

Should work in both msys, and normal bash.

Reply:
Actually that wont work on Mac, but 
ctest --output-on-failure `getconf _NPROCESSORS_ONLN` does seem to work on Mac/Win(msys)/Linux

Reply:
ah! Thanks 👍

Reply:
I'm ok with  a -j4 for now. We can fine-tune later.
We've had apparent problems with too much parallelism on the free CI systems, and have tuned down to j4 there. Who knows what the Kokoro machines are like.

Reply:
According to [kokoro documentation](https://g3doc.corp.google.com/devtools/kokoro/g3doc/development/gcp_ubuntu_config.md#os-kernel-cpu-memory), all the linux/mac/windows machines have 4 logical cores at the moment. But that'll likely increase in the future.